mypy/plugin.py

```diff
@@ -444,3 +500,14 @@ def int_pow_callback(ctx: MethodContext) -> Type:
         else:
             return ctx.api.named_generic_type('builtins.float', [])
     return ctx.default_return_type
+
+
+# This is an incredibly frumious hack. If this module is compiled by mypyc,
+# set the module 'Plugin' attribute to point to InterpretedPlugin. This means
+# that anything interpreted that imports Plugin will get InterpretedPlugin
+# while anything compiled alongside this module will get the real Plugin.
+if isinstance(int_pow_callback, types.BuiltinFunctionType):
```

Comment:
Is it possible to write this as something like `if MYPYC: ...` where `MYPYC` is the magic constant that is `False` in reality but treated as `True` by mypyc?


(Not need to do this now but will be useful in future I think.)

Reply:
That is something we should have but don't right now